fix(1901): screwdriver.cd/manualStartEnabled annotation (#455)
screwdriver-cd/screwdriver-cd.github.io: docs/user-guide/configuration/annotations.md

| screwdriver.cd/coverageScope | `pipeline` / `job` | When using the coverage plugin, this will set the scope for project creation. Default behavior depends on cluster configuration (e.g. `COVERAGE_SONAR_ENTERPRISE`), please check with your cluster admin for that. |
| screwdriver.cd/displayName | Job name to display in the pipeline graph | You can use any name you like for a job to appear in the pipeline graph, without being limited by the unique constraints of Yaml. |
| screwdriver.cd/mergeSharedSteps | `true` / `false` | When using a template, set it to `true`, and allow the user to merge steps defined in shared and job. Default is `false`. |

> | screwdriver.cd/manualStartEnabled | `true` / `false` | You will not be able to start in the UI from a job that has this setting is `false`. Default is `true`. |

Reviewer: ```suggestion
| screwdriver.cd/manualStartEnabled | `true` / `false` | You will not be able to start from a job in the UI when this is set to `false`. Default is `true`. |
```
